public/scripts/openai.js

```diff
@@ -402,23 +395,20 @@ async function prepareOpenAIMessages(name2, storyString, worldInfoBefore, worldI
 
         console.log(total_count);
 
-        for (const example of openai_msgs_example) {
-            // get the current example block with multiple user/bot messages
-            let example_block = example;
+        // each example block contains multiple user/bot messages
+        for (let example_block of openai_msgs_example) {
+            if (example_block.length == 0) { continue; }
+            example_block = [new_chat_msg, ...example_block];
 
-            for (let k = 0; k < example_block.length; k++) {
-                if (example_block.length == 0) { continue; }
```

Comment:
Useless, because if `example_block.length == 0`, the loop will not be entered. Moved it to the outer loop.